### PR TITLE
Result_in_csv

### PR DESCRIPTION
**Pull request by samu80**
This API export drives_candidate result in csv file contains
  - All candidate first_name, last_name, email, score in specific drive

### CODE REVIEW

**Review by somnath77 — APPROVED** [review]

File: spec/controllers/api/v1/results_controller_spec.rb [hunk 1]
```diff
@@ -34,4 +34,24 @@
       expect(response).to have_http_status(200)
     end
   end
+
+  describe 'GET#csv_result' do
+    before do
+      organization = create(:organization)
+      user = create(:user)
+      candidate = create(:candidate, first_name: 'abc', last_name: 'xyz', email: 'ab@gmail.com')
```
Comment (rahulknojha) [inline spec/controllers/api/v1/results_controller_spec.rb:42]: give some proper name, abc and xyz ?

Comment (samu80) [reply spec/controllers/api/v1/results_controller_spec.rb:42]: okay

File: spec/controllers/api/v1/results_controller_spec.rb [hunk 2]
```diff
@@ -34,4 +34,24 @@
...
+      candidate = create(:candidate, first_name: 'Kiran', last_name: 'Patil', email: 'kiran@gmail.com')
+      @drive = create(:drive, updated_by_id: user.id, created_by_id: user.id,
+                              organization: organization)
+      create(:drives_candidate, drive_id: @drive.id, candidate_id: candidate.id, score: 8)
+    end
+
+    it 'returns candidate result data in csv' do
+      get :csv_result, params: { drife_id: @drive.id }, format: :csv
+
+      expected_row = [['Email', 'kiran@gmail.com'], %w[First_name Kiran], %w[Last_name Patil], %w[Score 8]]
```
Comment (rahulknojha) [inline spec/controllers/api/v1/results_controller_spec.rb:51]: we can remove under_score from headers, as it is not readable for users

**Review by rahulknojha — DISMISSED** [review]

File: spec/controllers/api/v1/results_controller_spec.rb (hunk 2 above)
Comment (samu80) [reply spec/controllers/api/v1/results_controller_spec.rb:50]: ok

**Pull request merged**